example/ios/IoReactNativeSecureStorageExampleTests/SecureStorageTests.swift

```diff
@@ -0,0 +1,63 @@
+import XCTest
+import io_react_native_secure_storage
+
+// Tests which can be run on a simulator or pyshical device
+final class SecureStorageTests: XCTestCase {
```

Comment:
What about removing or updating `testRendersWelcomeScreen`. It's not related with this file. But we still have this test coming from the initial scaffold and it's not working anymore. What do you think?